packages/remote-replay-launcher/src/types.ts

```diff
@@ -30,6 +30,13 @@ export interface ExecuteRemoteTestRunOptions {
 
   environment: string;
   isLockable: boolean;
+
+  /**
+   * The ID of the pull request in the hosting provider, if one is available.
+   * The backend will try to automatically find one from the commit if this is not provided, but in some
+   * cases (e.g. PRs from forks on GitHub) this may not work, hence we prefer to set this if possible.
```

Comment:
I think we should still use the BE as primary source of truth for this
```suggestion
   * cases (e.g. PRs from forks on GitHub) this may not work, hence one can be provided here.
```